db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.6
--- Dumped by pg_dump version 9.5.6
+-- Dumped from database version 9.5.4
+-- Dumped by pg_dump version 9.5.4
```

Comment:
Please remove this - you haven't changed the schema at all so there's no need for these changes which just reflect your local environment being different.